dev/snippets/config/templates/freeform.tmpl

```diff
@@ -1,5 +1,4 @@
-/// Flutter code sample for {{element}}
-
```

Comment:
Was the removal of the blank line intentional? Should there maybe be an empty `//` line to separate this line a little bit from the description?

Reply:
Yes, it was intentional. Because of the blank line, the import sorter tried to separate the description and attach it to the first import listed, meaning that it would end up with code like this, which didn't make sense. Now it just leaves it alone. If you think the blank line still makes sense, I could add a `//` line between two two to keep it as one comment.

```dart
// Flutter code sample for Foo

import 'dart:ui';

// Sample description
import 'package:package/package.dart';
import 'package:package2/package2.dart';
```

Reply:
Not feeling strongly about this, but a `//` line separating the two would make sense to me.